Ships as Raw.txt

```diff
@@ -0,0 +1,780 @@
+Zoom 93% 
+
+Name|Gladiator I-class Star Destroyer
+Faction|Empire 
+Size|Small
+Points|56
+Hull|5
+Defense token|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Nav chart 
+Speed 1|2 
+Speed 2|1 1 
+Speed 3|0 1 1

```

Comment:
All rows should have key|value. This one doesn't.

We also don't have a Speed 4.  Now you and I know that a gladiator can't get to speed 4.  I'd like to propose 2 solutions here.

1) We can do something like Nav chart:2 11 011 -
It puts all the data in one row, at the expense of more difficulty parsing later.

OR

2) We drop the nav chart line and add in a line for Speed 4 like Speed 4: -

Reply:
Lets do number 2. I would say lets try 
Speed 1|2 
Speed 2|1 1 
Speed 3|0 1 1
Speed 4|-